### DISCUSSION TITLE
prep_baseline

### DISCUSSION THREAD

**Discussion by kjz1997** [isce-framework/isce2#860] [Q&A]
hello, when I was using iSCE to process data, I found a **prep_baseline** file with the same range as my processing range. Is this file the spatial baseline for each pixel? Or other content

**Comment by thescardo** [comment]
Sorry if this isn't helpful, but this tutorial helped me: https://github.com/isce-framework/isce2-docs/blob/master/Notebooks/UNAVCO_2020/TOPS/topsApp.ipynb

There's a lot of useful resources in that repo in general.
